src/main/kotlin/com/chattriggers/ctjs/minecraft/libs/renderer/Text.kt

```diff
@@ -138,7 +138,6 @@ class Text {
 
         for (i in 0 until maxLines) {
             if (i >= lines.size) break
-            // TODO: Pass in color param to Renderer.drawString?
             if (shadow) {
                 Renderer.getFontRenderer().drawWithShadow(Renderer.matrixStack.toMC(), lines[i], x ?: this.x, yHolder, color)
```

Comment:
This should just use `Renderer.drawString`

Reply:
So we can remove `Renderer.drawStringWithShadow` completely?